Spanner/src/Connection/Grpc.php

```diff
@@ -225,6 +225,14 @@ public function listInstances(array $args)
     public function getInstance(array $args)
     {
         $projectId = $this->pluck('projectId', $args);
+
+        $mask = [];
```

Comment:
We don't need to do this if there's no field mask given. It'll save us a few cycles.

```php
$mask = $this->pluck('fieldMask', $args, false);
if ($mask) {
    foreach () {}
    $args['fieldMask'] = [...];
}
```

Reply:
refactored